### ISSUE TITLE
Special case for deprecated comment

### ISSUE THREAD

**Issue by ammario** [codercom/depr#3]
The Deprecated comment begins after another word: https://gh.coder-internal.com/go/models/blob/develop/v2/containerresourceplans.table.gen.go#L25

**Comment by nhooyr** [comment]
Seems like a bug in models, the deprecated comment doesn't use a colon.

**Comment by ammario** [comment]
Are you saying that if it was `// comment: Deprecated: asdfasdfs. Asdfasfsdf` it would work?

**Comment by nhooyr** [comment]
Nah, it wouldn't work atm but that's what I think the convention should be.

**Comment by ammario** [comment]
sounds good
